__tests__/api.test.js

```diff
@@ -24,8 +24,30 @@ describe('GET requests', () => {
             expect(snacks[0]).toHaveProperty('description')
         })
     })
-=
-    test('200: responds with correct specific article object', () => {
+    test('200 : responds with correct articles object', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((result) => {
+            const {body} = result
+            const articles = body.articles
+            expect(body).toHaveProperty('articles')
+            expect(articles).toHaveLength(12)
+
+            articles.forEach((article) => {
+                expect(article).toHaveProperty('article_id');
```

Comment:
You can add a second argument to .toHaveProperty() which you can use to check the type of what has been returned. Have a look at expect.any on the jest docs - this can passed as a second argument to .toHaveProperty. This would be a much thorough test as it would not only check for keys, but would allow us to check the type of the value at that property.